integration_tests/test_domestic_migration.py

```diff
@@ -323,13 +330,12 @@ def test_addresses_during_moves(
             before_units[~mask_moved_units], after_units[~mask_moved_units]
         )
 
-        # Check that some movers are in a new state
-        # TODO Check that *most* are in a new state when we add all 50+ locations
-        if len(metadata.UNITED_STATES_LOCATIONS) > 1 and mask_moved_units.sum() > 1:
-            assert any(
+        # Check that most movers are in a new state
+        if number_states_in_sim > 1 and mask_moved_units.sum() > 1:
+            assert (
                 before_units.loc[mask_moved_units, state_id_col]
                 != after_units.loc[mask_moved_units, state_id_col]
-            )
+            ).mean() > 0.5  # TODO: pick a smarter number?
```

Comment:
What is the rationale for the 0.5?

Reply:
Not very great rationale - it's literally that most are changed states. I tried being smarter by doing something like making sure the mean > 0.9 * 50/51 (51 being the number of states in the US) but that was failing the test in some cases, eg in one time step only 7 businesses moved and one of them moved within state but 6/7 didn't make the cut

Reply:
@mattkappel Going to merge for now - happy to readdress if you have any better ideas! 

(or @zmbc since you mentioned maybe trying to think of a statistically better way to deal with some of these fuzzy asserts)